lib/signaling/v2/peerconnection.js

```diff
@@ -485,6 +486,37 @@ class PeerConnectionV2 extends StateMachine {
     return true;
   }
 
+  _shouldUpdateEncodings(track) {
+    if (track.kind !== 'video') {
+      return false;
+    }
+
+    if (util.guessBrowser() === 'safari') {
+      return true;
+    }
+
+    // NOTE(mpatwardhan):for chrome, update only non-screen share tracks, and only if adaptiveSimulcast is enabled.
```

Comment:
Just curious, what is the reason why?

Reply:
Because chrome does good job for screen share tracks - it keeps same dimensions but different frame rate for the simulcast layers. At one point we did configure the screen share track for chrome but found that it did not work as well (VIDEO-7000)

Reply:
Update: we decided to configure chrome screen share tracks as well. They will use different config than the camera tracks as described in  https://code.hq.twilio.com/client/sdk-frd/blob/jsdk_updates_for_adaptive_simulcast/video/adaptive-simulcast.md#configuring-simulcast